scripts/generateTypeScriptDefinitions.js

```diff
@@ -107,6 +119,31 @@ export async function generateTsDefsForJsGlobs(
         return;
       }
       try {
+        if (isModuleDeclarationOnly(source)) {
```

Comment:
this section does not lint, or prettify, or anything. I have a local refactor that turns this into a small "pipeline" of sorts with pre & post actions, since the difference between transforming source and module definitions fit cleanly in before & after while `translateFlowDefToTSDef` does the majority of the work.